admin/controllers/pupil-pin.js

```diff
@@ -25,7 +25,7 @@ const getGeneratePinsOverview = async function getGeneratePinsOverview (req, res
   res.locals.pinEnv = pinEnv
   res.locals.pageTitle = isLiveCheck ? 'Generate school passwords and PINs for the official check' : 'Generate passwords and PINs for the try it out check'
   req.breadcrumbs(res.locals.pageTitle)
-  const buttonText = isLiveCheck ? 'Generate official PINs' : 'Generate Try it out PINs'
+  const buttonText = isLiveCheck ? 'Generate school password and PINs for the official check' : 'Generate password and PINs for the try it out check'
```

Comment:
This may be desired, but I wanted to point out that the official check has the word 'school' and the TIO text doesn't.